src/Inertial_TEST.cc

```diff
@@ -603,3 +603,82 @@ TEST(Inertiald_Test, AdditionInvalid)
     EXPECT_TRUE((i0 + i).MassMatrix().IsValid());
   }
 }
+
+/////////////////////////////////////////////////
+TEST(Inertiald_Test, Subtraction){
+
+  // Subtract one half-cube from a full cube
+  { 
+    // full cube
+    const double mass = 12.0;
+    const math::Vector3d size(1, 1, 1);
+    math::MassMatrix3d cubeMM3;
+    EXPECT_TRUE(cubeMM3.SetFromBox(mass, size));
+    const math::Inertiald cube(cubeMM3, math::Pose3d::Zero);
```

Comment:
hmm, makes sense. I was also thinking to add some more unit tests for subtraction operator different from addition operator. Should I add them as well inside `AdditionSubtraction`

Reply:
I think the tests can be shared in most cases. Anywhere that you would test `a + b == c`, you can also test `c - a == b` and `c - b == a` and it would prevent a lot of duplicated code